2차/윤형진_약수의-합.py

```diff
@@ -0,0 +1,14 @@
+def solution(n):
+    answer = 0
+    
+    if n == 0 :
+        pass
+    elif n == 1 :
+        answer = 1
+    else :     
+        for i in range(2, n) :
+            if n % i == 0 :
+                answer = answer + i
+
+        answer = answer + 1 + n
```

Comment:
구조분해를 하는게 좋을 것 같습니다.
```suggestion
    if n <= 1 :
        return n

    for i in range(2, n) :
        if n % i == 0 :
            answer = answer + i

    answer = answer + 1 + n
```